docs/content/integrations/snowflake/reference.mdx

```diff
@@ -351,24 +442,23 @@ In this example, the `iris_dataset` asset will be stored in the `IRIS` schema, a
   </ol>
 </Note>
 
----
+### Storing timestamp data in Pandas DataFrames
 
-## Storing timestamp data in Pandas DataFrames
+Due to a longstanding [issue](https://github.com/snowflakedb/snowflake-connector-python/issues/319) with the Snowflake Pandas connector, loading timestamp data from a Pandas DataFrame to Snowflake sometimes causes the data to be corrupted. In order to store timestamp data properly, it must have a timezone attached.
```

Comment:
this has actually changed! I totally forgot to update the docs though. I think we just need the note now so that users on old versions know to look at the migration guide

Reply:
- For `old versions`, could you specify which version is the last that needs to look at the migration guide?
- Which migration guide? Is this documented already, or will it be a part of 1.7?

Reply:
I think we can just leave the note that starts on line 451. So the versions that need to worry about migration are anything below 0.19.0. And the correct section of the migration guide is linked in the note